### PR TITLE
added ringaction to NukiOpener

### PR DESCRIPTION
**Pull request by pree**
I've added the ringactionTimestamp & ringactionState to the NukiOpener.

This comes with the Bridge Firmware 2.7.0 & Opener Firmware 1.5.1, which are currently in Beta.
(REL: https://developer.nuki.io/t/bridge-beta-fw-2-7-x-1-17-x/6792)

### CODE REVIEW

File: pynuki/opener.py [hunk 1]
```diff
@@ -10,6 +10,14 @@ class NukiOpener(NukiDevice):
     def is_rto_activated(self):
         return self.state == const.STATE_OPENER_RTO_ACTIVE
 
+    @property
+    def ringactionTimestamp(self):
```
Comment (pschmitt) [inline pynuki/opener.py:14]: Please use snake case


```suggestion
    def ring_action_timestamp(self):
```

File: pynuki/opener.py [hunk 2]
```diff
@@ -10,6 +10,14 @@ class NukiOpener(NukiDevice):
...
+        return self._json.get("ringactionTimestamp")
+
+    @property
+    def ringactionState(self):
```
Comment (pschmitt) [inline pynuki/opener.py:18]:
```suggestion
    def ring_action_state(self):
```

**Review by pschmitt — CHANGES_REQUESTED** [review]
Almost lgtm ;)

File: pynuki/opener.py (hunk 1 above)
Comment (pree) [reply pynuki/opener.py]: done :)

**Pull request merged**